packages/vsce/README.md

```diff
@@ -184,6 +185,16 @@ If the applied filter results in more than 500 records, either change the filter
 <img src="/packages/vsce/docs/images/record-count-increment.png" alt="Zowe CICS Explorer Record Count Increment in Setting UI" width="700px"/>
 </p>
 
+### Inspect CICS resources
+
+View resource information without using the CICS tree. Access the command palette and filter for `Inspect CICS Resource`. You'll be prompted for your Zowe CICS profile, a resource type, and a resource name. The CICS Resource Inspector opens showing the details of the specified resource.
+
+**Tip:** Show the command palette using `ctrl`+`shift`+`p` for a keyboard-driven approach without switching to CEMT!
```

Comment:
I'm not sure about including this tip, since it's not really specific to the Zowe CICS extension, only shows the Windows keyboard shortcut, and users could have edited the command palette shortcut.